storage/innobase/buf/buf0dump.cc

```diff
@@ -58,7 +58,7 @@ take after being waked up. */
 static volatile bool	buf_dump_should_start;
 static volatile bool	buf_load_should_start;
 
-static bool	buf_load_abort_flag;
+static Atomic_relaxed<bool> buf_load_abort_flag;
```

Comment:
Are there any downsides to relaxed memory ordering for this flag especially with regards to other operations?